package.json

```diff
@@ -82,7 +81,10 @@
     "jest-canvas-mock": "^2.2.0",
     "jest-environment-jsdom-sixteen": "^1.0.3",
     "jest-prop-type-error": "^1.1.0",
+    "kleur": "^4.1.1",
```

Comment:
Should some of these be in the DevDependencies instead?

Reply:
🤔 They actually should be. I think something got lost when I force pushed.

Reply:
Ha, so we were looking at the collapsed diff view (notice the line numbers) 🤣